viadot/utils.py

```diff
@@ -483,4 +484,153 @@ def wrapper(*args, **kwargs) -> pd.DataFrame:
 
 
 def get_fqn(table_name: str, schema_name: str = None) -> str:
-    return f"{schema_name}.{table_name}" if schema_name else table_name
\ No newline at end of file
+    return f"{schema_name}.{table_name}" if schema_name else table_name
+
+
+def validations(
+    df: pd.DataFrame,
+    tests: dict = None,
```

Comment:
Nit: it would be better to have either an Enum or a model for this, which would allow IDE to display suggestions and documentation. Eg.

```python
from pydantic import BaseModel

class DataFrameTests(BaseModel):
    """Available DataFrame tests."""
    column_size: dict[str, int] | None
    column_unique_values: list[str]
    ...

def validate(..., tests: DataFrameTests = None):
    ...
```

(just FYI, no need to add it now)